configure.ac

```diff
@@ -78,7 +78,8 @@ m4_define([libva_lt_age],
 m4_define([libdrm_version], [2.4])
 
 # Wayland minimum version number
-m4_define([wayland_api_version], [1.0.0])
+# 1.11.0 for wl_proxy_create_wrapper
+m4_define([wayland_api_version], [1.11.0])
```

Comment:
The CI *does* use --enable-wayland in PR builds.  Unfortunately, libva's configure silently disables wayland if requirements aren't met :(.  That is a bug I guess.

For PR builds, the CI does not build downstream dependents (e.g. libva-utils) on top of the PR.  That only happens for post-merge CI builds right now (where we noticed the breakage, i.e. libva-utils expected libva-wayland et. al. to exist).

Nonetheless, I agree we need to keep moving forward... we'll figure out the best path to deal with this.  Thankfully, we build inside pre-configured docker containers so adding a pre-compiled wayland 1.11 to them might be the best solution for now.  But that's not your problem to worry about :)